tests/test_astrom.py

```diff
@@ -36,17 +36,17 @@ def test_astrom():
     expected_platescale = 21.8
     expected_northangle = 45
 
-    # check orientation is correct within 0.3 [deg]
+    # check orientation is correct within 0.05 [deg]
     # and plate scale is correct within 0.5 [mas] (arbitrary)
     assert astrom_cal.platescale == pytest.approx(expected_platescale, abs=0.5)
-    assert astrom_cal.northangle == pytest.approx(expected_northangle, abs=0.3)
+    assert astrom_cal.northangle == pytest.approx(expected_northangle, abs=0.05)
 
     # check that the center is correct within 30 [mas]
```

Comment:
This line should be changed to the following to match the change to the code:
`# check that the center is correct within 3 [mas]`